src/main.cpp

```diff
@@ -71,6 +72,13 @@ static int inner_main(int argc, char** argv)
     // execution return value
     int retVal = phantom->returnValue();
     delete phantom;
+
+#ifndef NDEBUG
```

Comment:
`_DEBUG` isn't defined.  I think what we actually want here is `#ifndef QT_NO_DEBUG`.